synapse/models/biz.py

```diff
@@ -17,12 +17,21 @@ def getModelDefs(self):
                 ('biz:stake', ('guid', {}), {
                     'doc': 'A stake or partial ownership in a company.',
                 }),
+                ('biz:listing', ('guid', {}), {
+                    'doc': 'A product or service being listed for sale at a given price by a specific seller.',
+                }),
                 ('biz:bundle', ('guid', {}), {
-                    'doc': 'Instances of a specific product offered for a price.',
+                    'doc': 'A bundle allows construction of products which bundle instances of other products.',
```

Comment:
Consider rewording if possible... definition is a little unclear.

Reply:
suggestions accepted :)